packages/destination-actions/src/destinations/klaviyo/upsertProfile/index.ts

```
@@ -188,6 +193,36 @@ const action: ActionDefinition<Settings, Payload> = {
 
       throw new APIError('An error occurred while processing the request', 400)
```

This nees to change. Could you just rethrow the error instead of overriding error code here? This might affect retries.